SleepEngine/Engine/Config/EngineConfigData.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <Engine/Utils/json_serialize_utils.h>
+
+BEGIN_NAMESPACE_SLEEP
+
+struct EngineConfigData
+{
+    Layer MaxLayer = 0;
+    size_t MeterLengthInPixels = 0;
+    glm::vec2 PrimaryWindowSize;
+};
```

Comment:
So every time we will add new value to a config we will also add it in this struct?

Reply:
Maybe we can find some way to not do it?

Reply:
I can't think of any
i definetely don't want to access it by string, so we need a struct